maas/plugins/galera_check.py

```diff
@@ -13,18 +13,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from __future__ import division
```

Comment:
another remnant from when I was calculating percentage in the plugin. Will remove